pkg/asset/manifests/content/bootkube/kube-system-secret-etcd-client.go

```diff
@@ -0,0 +1,25 @@
+package bootkube
+
+import (
+	"text/template"
+)
+
+var (
+	// KubeSystemSecretEtcdClient is the constant to represent contents of kube-system-secret-etcd-client.yaml file
+	KubeSystemSecretEtcdClient = template.Must(template.New("kube-system-secret-etcd-client.yaml").Parse(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: etcd-client
+  namespace: kube-system
+  labels:
+    tier: "control-plane"
+    k8s-app: "kube-apiserver"
+    openshift.io/control-plane: "true"
+    openshift.io/component: "api"
+type: SecretTypeTLS
+data:
+  tls.crt: {{ .EtcdClientCert }}
+  tls.key: {{ .EtcdClientKey }}
```

Comment:
With #587 landed, you'll have to restore these two [here][1] and back up the chain within `Manifests`.

[1]: https://github.com/openshift/installer/pull/587/files#diff-36a138ec46315c3afef6bca683543ee8L143

Reply:
Thanks for the heads up. Done.